influxdb/__init__.py

```diff
@@ -6,16 +6,20 @@
 from __future__ import print_function
 from __future__ import unicode_literals
 
+import os
+
 from .client import InfluxDBClient
-from .dataframe_client import DataFrameClient
 from .helper import SeriesHelper
 
 
 __all__ = [
     'InfluxDBClient',
-    'DataFrameClient',
     'SeriesHelper',
 ]
 
+if "INFLUXDB_NO_DATAFRAME_CLIENT" not in os.environ:
```

Comment:
This should probably be:
```
if "INFLUXDB_NO_DATAFRAME_CLIENT" not in os.environ or not os.environ["INFLUXDB_NO_DATAFRAME_CLIENT"]:
```

To allow for false-values to be used, like INFLUXDB_NO_DATAFRAME_CLIENT=0

Reply:
Unfortunately, since `os.environ` stores values as strings, `not os.environ["INFLUXDB_NO_DATAFRAME_CLIENT"]` will not be enough to check for falsey string values.

We could add a list of values that we count as falsey and check if the value is in there if you'd like to enable this kind of behavior. Something like:

```if os.environ.get("INFLUXDB_NO_DATAFRAME_CLIENT", "0").lower() not in ["0", "false", ""]``

Reply:
That would work, and I think the false values in your example should be sufficient